docker/files/run_faf

```diff
@@ -27,3 +27,5 @@ else
     sudo -u postgres /usr/bin/pg_ctl start -D /var/lib/postgres/data -w > /dev/null
     sudo -u faf faf-migrate-db
 fi
+crond
+init_db && apachectl -D FOREGROUND
```

Comment:
There is no init_db anymore, and "-D FOREGROUND" should be "-DFOREGROUND".

Reply:
Yeah, I forgot to push the fix with init_db. Fixed. And are you sure about the `-DFOREGROUND`? I think that separated works as well.

Reply:
Yes, it works, but you get "Passing arguments to httpd using apachectl is no longer supported." warning. -DFOREGROUND is special parameter, check man page of httpd.